src/pages/messages/components/NviCandidateActions.tsx

```diff
@@ -222,12 +226,15 @@ export const NviCandidateActions = ({ nviCandidate, nviCandidateQueryKey }: NviC
               variant="outlined"
               fullWidth
               size="small"
-              sx={{ mb: '1rem' }}
+              sx={{ mb: '1rem', bgcolor: '#eeeeee' }}
               disabled={isMutating || hasSelectedRejectCandidate}
+              endIcon={<ClearIcon />}
               onClick={() => setHasSelectedRejectCandidate(true)}>
               {t('tasks.nvi.reject_nvi_candidate')}
             </Button>
 
+            <Divider sx={{ mb: '1rem', bgcolor: 'lightgray' }} />
```

Comment:
Ser man egentlig forskjell? Hvis ja, burde alle dividers oppdateres?

```suggestion
            <Divider sx={{ mb: '1rem' }} />
```

Reply:
Den var ikke synlig uten å legge til `bgcolor`, av en eller annen grunn 🤔